mutacc_auto/parse/parse_housekeeper.py

```diff
@@ -0,0 +1,21 @@
+from mutacc_auto.commands.housekeeper_command import HousekeeperCommand
+
+def get_bams_from_housekeeper(housekeeper_output):
+
+    bam_files = {}
+
+    #CHECK FOR FILES WITH 'BAM' TAG AND STORE IN DICTIONARY
+    for row in housekeeper_output.split('\n'):
+
+        fields = row.split('|')
+
+        tags = fields[-1].split(',')
+
+        if len(tags) == 2 and tags[0].strip() == 'bam':
```

Comment:
Tags[0].strip() please store in a variable to explain it or add a comment so one knows what it represents